doc/sphinx/proof-engine/vernacular-commands.rst

```diff
@@ -381,48 +325,26 @@ Requests to the environment
       .. coqtop:: all
 
          SearchPattern (_ + _ = _ + _).
-
          SearchPattern (nat -> bool).
-
          SearchPattern (forall l : list _, _ l l).
 
-   Patterns need not be linear: you can express that the same expression
-   must occur in two places by using pattern variables `?ident`.
-
-
-   .. example::
-
       .. coqtop:: all
 
          SearchPattern (?X1 + _ = _ + ?X1).
 
-   .. cmdv:: SearchPattern @term inside {+ @qualid }
-
-      This restricts the search to constructions defined in the modules
-      named by the given :n:`qualid` sequence.
+.. cmd:: SearchRewrite @one_term {? {| inside | outside } {+ @qualid } }
 
-   .. cmdv:: SearchPattern @term outside {+ @qualid }
+   Displays the name and type of all hypotheses of the
+   selected goal (if any) and theorems of the current context whose
+   conclusion is an equality for which one side matches the
+   expression :n:`@one_term`.
 
-      This restricts the search to constructions not defined in the modules
-      named by the given :n:`qualid` sequence.
+   For a statement in the form :n:`{* P__i -> } C`, the :n:`P__n` are the *premises*
+   and :n:`C` is the *conclusion*.
```

Comment:
Until we can have these links to the definitions of standard notions that we talked about, I feel like this sentence is really to odd to be kept in the description of `SearchRewrite`.